src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx

```diff
@@ -1,23 +1,190 @@
+  //   return [['relays', `/organizations/${this.props.organization.slug}/relay-keys`]];
+  // }
+
+  handleDelete = (id: Relay['public_key']) => () => {};
+
+  handleAdd = () => {};
+
   renderBody() {
+    const {relays} = this.state;
     return (
       <React.Fragment>
         <SettingsPageHeader title={t('Relays')} />
+        <Panel>
+          <PanelHeader>{t('Relays')}</PanelHeader>
+          <PanelAlert type="info">
+            {tct('For more details, see [linkToDocs].', {
+              linkToDocs: (
+                <ExternalLink href={RELAY_DOCS_LINK}>
+                  {t('full Relay documentation')}
+                </ExternalLink>
+              ),
+            })}
+          </PanelAlert>
+          <PanelBody>
+            {relays.map(
+              ({public_key, name, created, last_seen, first_seen, last_modified}) => (
+                <Content key={public_key}>
+                  <Info>
+                    <InfoItem>
+                      <Name>{name}</Name>
+                    </InfoItem>
+                    <InfoItem>
+                      <PublicKey>{public_key}</PublicKey>
+                    </InfoItem>
+                    <InfoItem>
+                      <Time label={t('Created:')} date={created} />
+                    </InfoItem>
+                    <InfoItem>
+                      <Time label={t('First Seen:')} date={first_seen} />
+                    </InfoItem>
+                    <InfoItem>
+                      <Time label={t('Last Seen:')} date={last_seen} />
+                    </InfoItem>
+                    <InfoItem>
+                      <Time label={t('Last modified:')} date={last_modified} />
+                    </InfoItem>
+                  </Info>
+                  <Tooltip title={t('Delete Rule')}>
+                    <Button
+                      label={t('Delete Rule')}
+                      size="small"
+                      onClick={this.handleDelete(public_key)}
+                      icon={<IconDelete />}
+                    />
+                  </Tooltip>
+                </Content>
+              )
+            )}
+          </PanelBody>
+          <PanelAction>
+            <Button href={RELAY_DOCS_LINK} target="_blank">
+              {t('Read the docs')}
+            </Button>
+            <Button onClick={this.handleAdd} priority="primary">
+              {t('Add Relay')}
+            </Button>
+          </PanelAction>
+        </Panel>
       </React.Fragment>
     );
   }
 }
 
 export default Relays;
+
+const Content = styled('div')`
+  display: grid;
+  grid-template-columns: 1fr max-content;
+  align-items: center;
+  border-bottom: 1px solid ${p => p.theme.borderDark};
+  padding: ${space(1)} ${space(2)};
+  :last-child {
+    border-bottom: 0;
+  }
+`;
+
+const Info = styled('div')`
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: ${space(1)};
+  > *:nth-child(1),
+  > *:nth-child(2) {
+    grid-column: span 4;
+  }
```

Comment:
What does this do?

Reply:
the first and the second grid-cell will take 4 columns. I could also use grid-column-start: 1 and grid-column-end: -1

Reply:
Oh I see -- I didn't realize Name/PubKey was part of the grid.